app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,70 @@
+class PostsController < ApplicationController
+  def index
+    if params[:user]
```

Comment:
Nothing grave but by convention this should be `user_id` and not `user`.
You could also define as a variable to no call params each time
```ruby
user_id = params[:user_id]
```